src/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -525,10 +523,18 @@ private MemoryStream CreateMemoryStream(long maxBufferSize, out Exception error)
             return new LimitMemoryStream((int)maxBufferSize, 0);
         }
 
-        private byte[] GetDataBuffer(MemoryStream stream)
+        private ArraySegment<byte> GetDataBuffer(MemoryStream stream)
         {
-            // TODO: Use TryGetBuffer() instead of ToArray().
-            return stream.ToArray();
+#if NETNative
```

Comment:
Should this be `NETNative || NET46`? [MSDN](https://msdn.microsoft.com/en-us/library/system.io.memorystream.trygetbuffer%28v=vs.110%29.aspx?f=255&MSPPError=-2147217396) says it should be supported on both UWP and 4.6.